src/index.ts

```diff
@@ -1,150 +1,205 @@
 #!/usr/env/bin node
-try {
+    try {
 	// point errors at typescript, not javascript!
 	require("source-map-support").install()
-} catch {}
+    } catch {}
```

Comment:
Nit: Looks like we are mixing tabs and spaces here

Reply:
woops! formatters are fighting with eachother, ! nice catch, i'll fix my editor lol